test/acceptance/missing_required_column.test.js

```diff
@@ -0,0 +1,27 @@
+import { test, expect } from '@playwright/test'
+
+// a playwright test that uploads a file with a missing required column, and checks that the correct error message is displayed
+test('when the user uploads a file with a missing required column, the page correctly indicates there\'s an error', async ({ page }) => {
+  await page.goto('/')
+  await page.getByRole('button', { name: 'Start now' }).click()
+
+  await page.waitForURL('**/dataset')
+
+  await page.getByLabel('Conservation area dataset').check()
+  await page.getByRole('button', { name: 'Continue' }).click()
+
+  await page.waitForURL('**/upload')
+
+  const fileChooserPromise = page.waitForEvent('filechooser')
+  await page.getByText('Upload data').click()
+  const fileChooser = await fileChooserPromise
+  await fileChooser.setFiles('test/testData/conservation-area-errors.csv')
```

Comment:
If I were being really nitpicky it might be helpful if you could name the file with which errors were in it for other people looking at it later, e.g 'conservation-area-missing-reference(-error?).csv'

Reply:
there's only one file with errors that i have. and it has missing ref and invalid values

Reply:
okay fair